encryption.js

```diff
@@ -60,34 +60,37 @@ function joinIvAndData(iv, data) {
 
 /* Encryption using email, passphrase and content */
 function encrypt(passphrase, content) {
-  let initVector = new Uint8Array(ivLen);
+  const initVector = new Uint8Array(ivLen);
   crypto.getRandomValues(initVector);
 
   const passphraseKey = encoder.encode(passphrase);
   const data = encoder.encode(content);
 
+  // eslint-disable-next-line no-console
```

Comment:
I ended up adding this in a few places. Not thrilled about it, but I didn't want to be so bold that I deleted all the debugging `console.log()` statements.